reporters/GlobalSummaryReporter.js

```diff
@@ -483,6 +494,22 @@ class GlobalSummaryReporter {
     const blocksToLookBack = Math.ceil(lookbackTimeInSeconds / blockTimeInSeconds);
     return blocksToLookBack;
   };
+
+  _getTokenHolders = async () => {
+    // TODO: This is a fragile implementation that scrapes etherscan's token holder page. It would likely fail if the
```

Comment:
Not trying to throw a wrench into this, but would it be acceptable (at least as a first pass) to just pull all the Transfer events down instead and tally each user's cumulative balance? My guess is that there wouldn't be too many events to parse since these are uniswap liquidity tokens.

Also cool keeping this as is. I agree that it's somewhat fragile and we'll need a better solution long term, but if it works, it works :).

Reply:
What do you mean by "these are uniswap liquidity tokens"?

I think this is pretty reasonable actually, since we already get synthetic `Transfer` events to track "burns", so we can just grab all "Transfers" once.

Reply:
However once there are > 1000 events this will require more than one call to `contract.getPastEvents` so this would require more complex logic to be scalable.

Reply:
How about for now we can attempt to scrape from etherscan, but if that fails for any reason then we can do the more inefficient querying of all token holders. Would also be helpful to have that logic somewhere anyways to track all token holders.

Reply:
https://infura.io/docs/ethereum/json-rpc/eth-getLogs I think that's an infura limitation, and I think it's 10,000. I'm going to look into it, but I think we can probably make the size unlimited using our own node.

Reply:
Agreed on the general approach. Let's keep the Etherscan method for now and reconsider if/when it breaks.

Reply:
I just realized: if we want to display "cumulative" token holder count, then we actually need to maintain our own list of token holders. As cool as it is scrape it off the website, I might have to deprecate that and query all `Transfer` events anyways.

Reply:
Done.